### ISSUE TITLE
Simple setting to change background color

### ISSUE THREAD

**Issue by Mr-Personality** [audacity/audacity#7208]
### Your idea

I recently upgraded to the latest Audacity and was delighted to find the interface is now way more attractive and streamlined. Thank you, developers!

**Problem:** I really love the "Light" color theme, but I just felt the background color was way too dark. I wanted to change it to something brighter such as #7D82A2. That's the only change I would make. For a "light" theme, this dark color just seemed too heavy.

Unfortunately I found that the only way to change this 1 color is to create my own whole new theme, and that is a very complicated process which first involves FINDING this guide: https://audacity.gitbook.io/dev/scripting/creating-custom-themes, then following the many steps toward creating my own theme. This is way too much work for such a simple change.

**Solution**: It would be nice to allow users to manually customize the background color, choosing between "theme default" or a color picker. OR simply a 'lightness' slider for the background color. Other prominent colors could also be customizable (such as the waveform color), but the background color is the most prominent and occupies most of the screen.

### Problem to be solved

See above.

**Comment by DavidBailes** [comment]
> I really love the "Light" color theme, but I just felt the background color was way too dark.

I agree - see: https://github.com/audacity/audacity/discussions/6845

**Comment by Mr-Personality** [comment]
Ah, glad I'm not alone in thinking this!
